Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -97,6 +97,15 @@ open class Player: BaseObject {
         core?.on(Event.didChangeActivePlayback.rawValue) { [weak self] _ in self?.bindPlaybackEvents() }
         core?.on(InternalEvent.userRequestEnterInFullscreen.rawValue) { [weak self] (info: EventUserInfo) in self?.trigger(Event.requestFullscreen.rawValue, userInfo: info) }
         core?.on(InternalEvent.userRequestExitFullscreen.rawValue) { [weak self] (info: EventUserInfo) in self?.trigger(Event.exitFullscreen.rawValue, userInfo: info) }
+        bindMediaControlEvents()
+    }
+    
+    private func bindMediaControlEvents() {
+        let eventsToListen: [Event] = [.willShowMediaControl, .didShowMediaControl, .willHideMediaControl, .didHideMediaControl]
```

Comment:
No strong opinion, but this has a few less chars
```suggestion
        let eventsToListen = [Event.willShowMediaControl, .didShowMediaControl, .willHideMediaControl, .didHideMediaControl]
```